<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenMenuBar</class>
 <widget class="QWidget" name="GenMenuBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>72</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="GenPushButton" name="action1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background: transparent;
    border-width: 0px;
    color: rgb(255,255,255);
	border-style: none;
}

QPushButton::pressed
{
    border-width: 0px;
	border-style: none;
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop:0 rgba(142, 207, 81, 255), stop:1 rgba(62, 125, 41, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/cutebox.qrc">
        <normaloff>:/images/add_new_icon.png</normaloff>:/images/add_new_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GenPushButton" name="action2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background: transparent;
    border-width: 0px;
    color: rgb(255,255,255);
	border-style: none;
}

QPushButton::pressed
{
    border-width: 0px;
	border-style: none;
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop:0 rgba(142, 207, 81, 255), stop:1 rgba(62, 125, 41, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/cutebox.qrc">
        <normaloff>:/images/download_icon.png</normaloff>:/images/download_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GenPushButton" name="action3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background: transparent;
    border-width: 0px;
    color: rgb(255,255,255);
	border-style: none;
}

QPushButton::pressed
{
    border-width: 0px;
	border-style: none;
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop:0 rgba(142, 207, 81, 255), stop:1 rgba(62, 125, 41, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/cutebox.qrc">
        <normaloff>:/images/copy_icon.png</normaloff>:/images/copy_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GenPushButton" name="action4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background: transparent;
    border-width: 0px;
    color: rgb(255,255,255);
	border-style: none;
}

QPushButton::pressed
{
    border-width: 0px;
	border-style: none;
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop:0 rgba(142, 207, 81, 255), stop:1 rgba(62, 125, 41, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/cutebox.qrc">
        <normaloff>:/images/settings_icon.png</normaloff>:/images/settings_icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenPushButton</class>
   <extends>QPushButton</extends>
   <header>genpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/cutebox.qrc"/>
 </resources>
 <connections/>
</ui>
